packages/completer/src/default/inlinehistoryprovider.ts

```diff
@@ -72,6 +72,11 @@ export class HistoryInlineCompletionProvider
 
     const multiLinePrefix = request.text.slice(0, request.offset);
     const linePrefix = multiLinePrefix.split('\n').slice(-1)[0];
+    const multiLineSuffix = request.text.slice(
+      request.offset,
+      request.text.length
+    );
+    const suffix = multiLineSuffix.split(' ')[0];
```

Comment:
It is not obvious why it splits on a whitespace and not on new line.

Reply:
If there is suffix, should we remove it from suggestion?

https://github.com/jupyterlab/jupyterlab/blob/6e1174e202b6bf5a7b725d59b5ddafc5f0afbed2/packages/completer/src/default/inlinehistoryprovider.ts#L131-L140